samples/regex_rewrite/BUILD

```diff
@@ -0,0 +1,42 @@
+load("//:plugins.bzl", "proxy_wasm_plugin_cpp", "proxy_wasm_plugin_rust")
+
+licenses(["notice"])  # Apache 2
+
+proxy_wasm_plugin_rust(
+    name = "plugin_rust.wasm",
+    srcs = ["plugin.rs"],
+    deps = [
+        "//cargo:regex",
+        "@proxy_wasm_rust_sdk//:proxy_wasm",
+        "@proxy_wasm_rust_sdk//bazel/cargo:log",
+    ],
+)
+
+# Blocked on emscripten update to fix Abseil/RE2 compatibility:
+# https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/pull/157
+#proxy_wasm_plugin_cpp(
```

Comment:
I went back and forth on this. I decided to leave it in because customers _can_ build the C++ plugin by using the linked ProxyWasmCppSdk PR. I added a note and confirmed this just now (and restored the absl version upgrade).